{"name": "test_with_invalid_credentials", "status": "passed", "start": 1718613577336, "stop": 1718613578570, "uuid": "25b5fc40-6fe3-463b-8ba7-66c365210a04", "historyId": "e930fbf7432442d4b6934232374e6f70", "testCaseId": "e930fbf7432442d4b6934232374e6f70", "fullName": "test_Login.TestLogin#test_with_invalid_credentials", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup_and_teardown', 'log_on_failure')"}, {"name": "suite", "value": "test_Login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "LAPTOP-ESL8NRHA"}, {"name": "thread", "value": "13652-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_Login"}]}